src/widgets/String.vue

```diff
@@ -2,60 +2,41 @@
 <template>
 	<record-field :required="required" :wrapped="false" :header="!inArray">
 		<title-component slot="title" :title="uiLabel" />
-		<div slot="header-right" class="header__right">
-			<InfoIcon :description="uiDescription"/>
-		</div>
+		<small slot="help" class="text-muted">
+			{{ uiDescription }}
+		</small>
+
+		<div slot="input">
+			<b-form-group
+				:label-cols="inArray ? 1 : ((makeLabel !== uiLabel) ? 3 : 0)"
+				:label-for="inArray ? 'input-' + property.toString() : property">
+				<span slot="label" v-if="makeLabel !== uiLabel">
+					{{ makeLabel }}
+					<delete-button v-if="inArray" slot="label" @click="deleteMe"/>
+				</span>
 
-		<div slot="input" class="input">
-			<p v-if="inArray" class="input__number">#{{property}}</p>
-			<b-form-input
-				:type="inputType"
-				:placeholder="uiPlaceholder"
-				:value="value"
-				:state="isValid ? null : false"
-				@input.native="updateValue">
-			</b-form-input>
-			<span class="input__delete">
-				<delete-button v-if="inArray" @click="deleteMe" />
-			</span>
+				<b-input-group>
+					<b-form-input
+						:id="inArray ? 'input-' + property.toString() : property"
+						:type="inputType"
+						:placeholder="uiPlaceholder"
+						:value="value"
+						:state="isValid ? null : false"
+						@input.native="updateValue">
+					</b-form-input>
+					<b-input-group-append>
```

Comment:
Delete button?

Reply:
delete button is part of the parent which is the Array.
here is a clear button instead, not a delete.

Reply:
removed